packager/debugger.html

```diff
@@ -27,6 +27,11 @@
   }
 };
 
+// Alias native implementations needed by the debugger before platform-specific
+// implementations are loaded into the global namespace
+window.debuggerSetTimeout = window.setTimeout;
```

Comment:
you can actually declare those as normal vars, no need to attach them to global context